### ISSUE TITLE
Issue setting defaultValue for multiple mui-rte on a single component

### ISSUE THREAD

**Issue by zwelethu** [niuware/mui-rte#183]
## Checklist

- [ ] This is not covered in the [repository examples](https://github.com/niuware/mui-rte/tree/master/examples).
- [ ] I am using the package [latest version](https://github.com/niuware/mui-rte/releases/latest).
- [ ] This is a bug report or an enhancement proposal.

## Description

Hi Everyone,

I am not sure if I missed something in the docs but I have an issue setting different defaultValues on the mui-rte when there is more than one in a component. If I set all the mui-rte to the same value then there is no issue. If I set them to different values then the page goes into an infinite loop. Please see my code below and please assist on how to get around this issue.
```
<MuiThemeProvider theme={defaultTheme}>    
<MUIRichTextEditor label="Macroscopic" 
    onChange={(data) =>  handleChange(data, 'marcroscopic') }
    defaultValue={macroscopic}
   />
</MuiThemeProvider>
<MuiThemeProvider theme={defaultTheme}>
    <MUIRichTextEditor label="Diagnosis" 
    onChange={(data) => handleChange(data, 'microscopic') }    
    defaultValue={diagnosis}    />
</MuiThemeProvider>
Any help is much appreciated.

**Comment by niuware** [comment]
The loop should not be happening on this [release](https://github.com/niuware/mui-rte/releases/tag/1.25.0). Can you verify?

**Comment by zwelethu** [comment]
Hi Erik,
I have tested it and the issue only occurs if there is an onChange event. Without an onChange event, there is no infinite loop. Is there any other way I can implement the onChange event?
Regards,Zweli

    On Thursday, October 8, 2020, 6:52:20 AM GMT+2, Erik Lopez <notifications@github.com> wrote:  
 
 


The loop should not be happening on this release. Can you verify?

**Comment by niuware** [comment]
This should be fixed on version `1.26.0` after removing the auto focus of the editor.
